src/templates/globals/medallia/index.tsx

```diff
@@ -0,0 +1,38 @@
+import Script from 'next/script'
+import { useEffect } from 'react'
+import {
+  getSurveyNumber,
+  loadForm,
+  onMedalliaLoaded,
+  setWindowVaSurvey,
+} from '@/lib/utils/medallia'
+
+export function MedalliaAssets() {
+  const scriptId = process.env.BUILD_TYPE === 'production' ? 2 : 5
+
+  useEffect(() => {
+    if (process.env.NEXT_PUBLIC_BUILD_TYPE === 'localhost') {
```

Comment:
do we need to put this into `next.config.js`'s  `env` section, or just handle that in the ticket for environment handling?

Reply:
https://nextjs.org/docs/pages/api-reference/next-config-js/env

Reply:
yeah, I think we should remove them and update to the newer approach. Good callout.